src/PointwiseFunctions/AnalyticData/GrMhd/AnalyticData.hpp

```diff
@@ -0,0 +1,12 @@
+// Distributed under the MIT License.
```

Comment:
Newbie question: can you please help me understand the purpose of this file?

Reply:
Because namespaces are used in many files, it is not obvious which file should be responsible for providing the namespace's documentation. Making a little header file just for the documentation makes this well-defined.